### ISSUE TITLE
Turn off formating for Project Description

### ISSUE THREAD

**Issue by darioo** [a-ono/redmine_per_project_formatting#7]
In previous version (0.0.2) when I uncheck "Apply project-wide formatting" I also turn off formating for Project Description. 

Now it is still turn on end on project "Overview" page I get un-rendered HTML codes eg.

<p>some description</p> 

Could you tell me how to solve this issue?

**Comment by maxiv** [comment]
Yes, Luxure Button and favorite projects, and mail in RedmineCRM Invoice plugin fails.
